integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSession.scala

```diff
@@ -101,8 +100,8 @@ class CarbonSession(@transient val sc: SparkContext,
           } catch {
             case e: Exception =>
               logError(String.format(
```

Comment:
It is in scala here, why still use `String.format`? Just use s"....$var..."

Reply:
ok, I will change